app/assets/stylesheets/globals.sass

```diff
@@ -15,6 +15,7 @@
 @import "base/pres"
 @import "base/typography"
 @import "base/icons"
+@import "base/map"
```

Comment:
alphebetise these, i think icons got chucked on the end too

Reply:
probably not a good idea if they are order dependent?

Reply:
its for code organisation reasons not anything technical :) otherwise as the file grows it can be hard to find an include